joomla/joomla-cms
# [4.0] Fix the html incompatible paths

Pull Request for Issue #20832 .

### Summary of Changes
Paths were hardcoded and working only for root installation


### Testing Instructions
Apply patch

Change line https://github.com/joomla/joomla-cms/blob/c7b4d42ebf2243564728350ea21ed36689a9ba05/index.php#L16
to `define('JOOMLA_MINIMUM_PHP', '9.0');`

check the site
### Expected result
No errors


### Actual result
Script error



![screen shot 2018-06-28 at 14 32 06](https://user-images.githubusercontent.com/3889375/42034584-ccbb6cca-7ae0-11e8-8cd3-b67ffe47a0e0.png)
![screen shot 2018-06-28 at 14 31 49](https://user-images.githubusercontent.com/3889375/42034585-ccd58380-7ae0-11e8-93c5-cb1b7ed64018.png)

### Documentation Changes Required
No, bug fix


@mbabker @SniperSister are you ok with this approach?
